### ISSUE TITLE
When sending via QR code scanner, the amount is reset to zero.

### ISSUE THREAD

**Issue by eviltofu** [AlphaWallet/alpha-wallet-ios#5750]
When the QR code scanner is used to scan the address, the amount sent is changed from 0.001 to 0.

https://user-images.githubusercontent.com/1050309/200492750-69ee96ec-0cc7-49d8-915e-261432ab19bd.mov

**Comment by hboon** [comment]
Yeah, would be good to not reset the amount but only because the QR code is an address. If it's an EIP681 link the amount should be reset.

**Comment by oa-s** [comment]
should be fixed with latest version

**Comment by hboon** [comment]
Yeah, works for me too. Thanks
